notebooks/20220512_initial_sophie_exploration.ipynb

```diff
@@ -0,0 +1,269 @@
+{
```

Comment:
<p>So the genes that you're showing in this upset plot are the same as the genes in the blue bar from the first plot (<code>sig_&amp;_zscore</code>)from the first plot, right?</p><p> </p><p>If you want you could add a title so that context doesn't get lost if you use this plot outside of the NB, maybe something like "Overlap between genes meeting significance/z-score thresholds" would work.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/greenelab/2022-cf-sputum/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>